src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -215,6 +215,8 @@
       <_EmccCFlags Include="-DINVARIANT_GLOBALIZATION=1"       Condition="'$(InvariantGlobalization)' == 'true'" />
       <_EmccCFlags Include="-DLINK_ICALLS=1"                   Condition="'$(WasmLinkIcalls)' == 'true'" />
       <_EmccCFlags Include="-DENABLE_AOT_PROFILER=1"           Condition="$(WasmProfilers.Contains('aot'))" />
+      <_EmccCFlags Include="-DENABLE_BROWSER_PROFILER=1"       Condition="$(WasmProfilers.Contains('browser'))" />
```

Comment:
Use a `WasmProfiler` item instead of a property.

Reply:
`WasmProfilers` is a property, not item group. Do you suggest we change it ? It's already used by Uno.